templates/site/test-button.html

```diff
@@ -0,0 +1,46 @@
+{% extends "layouts/main.html" %}
+{% set page_title = "Test Buttons" %}
+
+{% block content %}
+    
+    {% with med_action_button = true%}
```

Comment:
why not have a `button_type` variable instead of setting variables to true? Wouldn't that be more readable?